src/system.rs

```diff
@@ -43,22 +44,67 @@ impl PendingTypes {
 
     /// Adds all the types from Context into given TypeStore and clears Context types for loading next set of types
     /// this is used after a schema named type/root type is loaded entirely into Context
-    pub fn update_type_store(&mut self, type_store: &mut TypeStore) -> IonSchemaResult<()> {
+    pub fn update_type_store(
+        &mut self,
+        type_store: &mut TypeStore,
+        load_isl_import: Option<&IslImport>,
```

Comment:
Could you add a description for the parameters to the doc comment for this function?